addOns/requester/src/main/java/org/zaproxy/zap/extension/requester/ManualRequestEditorPanel.java

```diff
@@ -184,11 +185,19 @@ protected JButton getBtnSend() {
             btnSend = new JButton();
             btnSend.setText(Constant.messages.getString("manReq.button.send"));
             btnSend.setEnabled(isSendEnabled);
+            btnSend.setMnemonic(KeyEvent.VK_ENTER);
+            btnSend.setToolTipText(getBtnSendTooltip());
             btnSend.addActionListener(e -> sendButtonTriggered());
         }
         return btnSend;
     }
 
+    protected static String getBtnSendTooltip() {
+        return !Constant.isMacOsX()
```

Comment:
I’m not sure why I used negative logic here, but it could be swapped

Reply:
Done.